sp/src/game/server/ai_speech_new.h

```diff
@@ -206,11 +206,11 @@ class CAI_Expresser : public ResponseRules::IResponseFilter
 
 	// --------------------------------
 	
-	bool CanSpeakConcept( AIConcept_t concept );
-	bool SpokeConcept( AIConcept_t concept );
-	float GetTimeSpokeConcept( AIConcept_t concept ); // returns -1 if never
-	void SetSpokeConcept( AIConcept_t concept, AI_Response *response, bool bCallback = true );
-	void ClearSpokeConcept( AIConcept_t concept );
+	bool CanSpeakConcept( const AIConcept_t &concept );
```

Comment:
These aren't build-fixes. I just thought if an `AIConcept_t` copy is avoided above via passing by reference, maybe it's useful here as well..?